example/lib/main.dart

```diff
@@ -1,260 +1,326 @@
 import 'package:example/app_colors.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
 import 'package:flutter_credit_card/credit_card_brand.dart';
 import 'package:flutter_credit_card/flutter_credit_card.dart';
 
-void main() => runApp(MySample());
+void main() => runApp(const MySample());
 
 class MySample extends StatefulWidget {
+  const MySample({Key? key}) : super(key: key);
+
   @override
-  State<StatefulWidget> createState() {
-    return MySampleState();
-  }
+  State<StatefulWidget> createState() => MySampleState();
 }
 
 class MySampleState extends State<MySample> {
+  bool isLightTheme = false;
   String cardNumber = '';
   String expiryDate = '';
   String cardHolderName = '';
   String cvvCode = '';
   bool isCvvFocused = false;
   bool useGlassMorphism = false;
   bool useBackgroundImage = false;
-  OutlineInputBorder? border;
+  bool useFloatingAnimation = true;
+  final OutlineInputBorder border = OutlineInputBorder(
+    borderSide: BorderSide(
+      color: Colors.grey.withOpacity(0.7),
+      width: 2.0,
+    ),
+  );
   final GlobalKey<FormState> formKey = GlobalKey<FormState>();
 
-  @override
-  void initState() {
-    border = OutlineInputBorder(
-      borderSide: BorderSide(
-        color: Colors.grey.withOpacity(0.7),
-        width: 2.0,
-      ),
-    );
-    super.initState();
-  }
-
   @override
   Widget build(BuildContext context) {
+    SystemChrome.setSystemUIOverlayStyle(
+      isLightTheme ? SystemUiOverlayStyle.dark : SystemUiOverlayStyle.light,
+    );
     return MaterialApp(
       title: 'Flutter Credit Card View Demo',
       debugShowCheckedModeBanner: false,
+      themeMode: isLightTheme ? ThemeMode.light : ThemeMode.dark,
       theme: ThemeData(
-        primarySwatch: Colors.blue,
+        brightness: Brightness.light,
+        primaryColor: Colors.white,
+        scaffoldBackgroundColor: Colors.black,
+      ),
+      darkTheme: ThemeData(
+        brightness: Brightness.dark,
+        primaryColor: Colors.black,
+        scaffoldBackgroundColor: Colors.white,
       ),
       home: Scaffold(
         resizeToAvoidBottomInset: false,
-        body: Container(
-          decoration: const BoxDecoration(
-            image: DecorationImage(
-              image: ExactAssetImage('assets/bg.png'),
-              fit: BoxFit.fill,
-            ),
-            color: Colors.black,
-          ),
-          child: SafeArea(
-            child: Column(
-              children: <Widget>[
-                const SizedBox(
-                  height: 30,
+        body: Builder(
+          builder: (BuildContext context) {
+            final Color bgColor = Theme.of(context).scaffoldBackgroundColor;
+            return Container(
+              decoration: BoxDecoration(
+                color: bgColor,
+                image: DecorationImage(
+                  image: ExactAssetImage(
+                    isLightTheme ? 'assets/bg-white.png' : 'assets/bg.png',
```

Comment:
convert assets/bg.png > assets/bg-dark.png and assets/bg-white.png > assets/bg-light.png